pylintrc

```diff
@@ -216,7 +217,7 @@ single-line-if-stmt=no
 no-space-check=trailing-comma,dict-separator
 
 # Maximum number of lines in a module
-max-module-lines=1000
+max-module-lines=1100
```

Comment:
It is. Most of the code that broke this rule is the enforcement code. I can open an issue to track shrinking it's size.